lib/kolekti/memory_persistence_strategy.rb

```diff
@@ -11,12 +11,16 @@ def initialize
     end
 
     def create_tree_metric_result(metric_configuration, module_name, value, granularity)
-      @tree_metric_results << {
-        metric_configuration: metric_configuration,
-        module_name: module_name,
-        value: value,
-        granularity: granularity
-      }
+      if @tree_metric_results.any? {|item| (item[:module_name] == module_name || item[:metric_configuration] == metric_configuration)}
```

Comment:
By https://github.com/mezuro/kolekti/issues/19 this should be a `&&` instead of `||`.